pkg/cmd/render/render.go

```diff
@@ -221,6 +221,10 @@ func newTemplateData(opts *renderOpts) (*TemplateData, error) {
 		return nil, err
 	}
 
+	if templateData.Platform == "Libvirt" {
+		templateData.setUnsupportedKeyForCR("/manifests/0000_12_etcd-operator_01_operator.cr.yaml")
```

Comment:
I don't understand why all libvirt clusters are assumed to need the etcd workaround for clusters with less than three nodes? There is nothing in the libvirt support which mandates this, is there?